js/gem-notice.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
I don't like that we have to do so much for this. Seems like it should be simpler...